common/util/parse.ts

```diff
@@ -49,6 +49,16 @@ export function parseWordsAsTags(text: string) {
   return parseTags(taggedText)
 }
 
+// TODO: fuzzy matching
+export const wordIn = (word: string, corpus: string) =>
+  corpus.toLocaleLowerCase().includes(word.toLocaleLowerCase())
+
+const checkAgainstQuery = (query: string, corpus: string) =>
+  query.split(' ').every((word) => wordIn(word, corpus))
```

Comment:
Concerned that for every word, corpus is being lowercased again. Probably worth special casing this by re-implementing wordIn again here so that corpus only gets lowercased once.

Reply:
I'd be very surprised if people do searches with more than like 5 words